pulsar-broker/src/main/java/org/apache/pulsar/PulsarStandalone.java

```diff
@@ -394,9 +392,7 @@ void createNameSpace(String cluster, String publicTenant, NamespaceName ns) thro
         }
 
         if (!nsr.namespaceExists(ns)) {
-            Policies nsp = new Policies();
-            nsp.replication_clusters = Collections.singleton(config.getClusterName());
```

Comment:
Why not fix it by something like:

```java
nsp.bundles = /* ... */;
```

It would be more clear for what's the default bundle policy of the standalone and not affected by the PulsarAdmin implementations.